extensions/wikia/PhalanxII/services/PhalanxService.class.php

```diff
@@ -65,13 +71,14 @@ public function __call($name, $args) {
 	 * check service status
 	 */
 	public function status() {
-		return $this->sendToPhalanxDaemon( "status", array() );
+		return $this->sendToPhalanxDaemon( "status", [] );
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "status" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)